warehousecontroller/mission2goal-translation-design.txt

```diff
@@ -0,0 +1,13 @@
+Mission Goal:
+	[TYPE, any/blue/green/red, ALL/[num], CURRENT/shelf1/shelf2/.., cbOut1/cbOut2/.]
```

Comment:
So this is the whole definition for the Mission I you would need to generate the PDDL goal below?